eks/ng/clusterautoscaler.go

```diff
@@ -0,0 +1,454 @@
+package ng
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"github.com/aws/aws-k8s-tester/pkg/fileutil"
```

Comment:
We should move this to `eks/ng/autoscaler`

Reply:
And automatically installed when `NG` has `"autoscaler":{"enable":true...` field